README.md

```diff
@@ -15,8 +15,22 @@ Plug 'paretje/deoplete-notmuch', {'for': 'mail'}
 ```
 
 ## Configuration
+
 ```vim
 " notmuch address command to fetch completions
-" NOTE: --format=sexp is required
-let g:deoplete#sources#notmuch#command = ['notmuch', 'address', '--format=sexp', '--output=recipients', '--deduplicate=address', 'tag:sent']
+" NOTE: --format=json is required
+let g:deoplete#sources#notmuch#command = ['notmuch', 'address', '--format=json', '--deduplicate=address', '*']
+```
+
+The command must output JSON in the following schema to be parseable by this plugin (so as long as the output is in spec, one may try to use a different source than `notmuch`):
+```json
+[
+    {
+        "name":"Jane Doe",
+        "address":"jane@mail.doe",
+        "name-addr":"Jane Doe <jane@mail.doe>"
+    },
+]
 ```
+
+You can modify the `notmuch search` to great extents, for example replace `'*'` by `tag:sent` to narrow the completions down to people who already received an email from you. For further information, see `man 1 notmuch-address` and `man 7 notmuch-search-terms`.
```

Comment:
One small remark: `tag:sent` also requires the option `--output=recipients`.

Reply:
Thanks, should be resolved now